common/rpc/interceptor/namespace.go

```diff
@@ -24,16 +24,37 @@
 
 package interceptor
 
+import (
+	"go.temporal.io/server/common/cache"
+)
+
 type (
-	NamespaceGetter interface {
+	NamespaceNameGetter interface {
```

Comment:
Can you add a small comment here saying something like "gRPC method `request` param must implement either `NamespaceNameGetter` or `NamespaceIDGetter` for namespace specific metrics to be reported properly."

Later on, people will add new APIs and it is not clear that there is this convention.

Reply:
done